ADT/sftp_oag_client_maytech.py

```diff
@@ -82,23 +95,24 @@ def main():
     # Main
     os.chdir('/ADT/scripts')
     oaghistory = gdbm.open('/ADT/scripts/oaghistory.db','c')
-    if not os.path.exists(download_dir):
-        os.makedirs(download_dir)
-    if not os.path.exists(staging_dir):
-        os.makedirs(staging_dir)
-    if not os.path.exists(quarantine_dir):
-        os.makedirs(quarantine_dir)
+    if not os.path.exists(DOWNLOAD_DIR):
+        os.makedirs(DOWNLOAD_DIR)
+    if not os.path.exists(STAGING_DIR):
+        os.makedirs(STAGING_DIR)
+    if not os.path.exists(QUARANTINE_DIR):
+        os.makedirs(QUARANTINE_DIR)
 
     # Note: do not archive the files - the OAG Import script will do the archiving
 
     downloadcount = 0
-    logger.debug("Connecting via SSH")
-    ssh = ssh_login(ssh_remote_host_maytech, ssh_remote_user_maytech, ssh_private_key)
-    logger.debug("Connected")
+    uploadcount = 0
+    logger.info("Connecting via SSH")
+    ssh = ssh_login(SSH_REMOTE_HOST_MAYTECH, SSH_REMOTE_USER_MAYTECH, SSH_PRIVATE_KEY)
+    logger.info("Connected")
     sftp = ssh.open_sftp()
 
     try:
```

Comment:
This `try` has no `except` to catch any exceptions

Reply:
It's in Line 147

Reply:
Ah! GitHub had hidden/collapsed that part